.github/workflows/nodejs.yml

```diff
@@ -21,6 +23,10 @@ jobs:
       run: |
         npm install
         npm run build --if-present
-        git diff --exit-code
+        git config user.name ${GITHUB_ACTOR}
+        git config user.email ${GITHUB_ACTOR}
```

Comment:
Can't you give the email address here?

Reply:
I first have to figure out how to get at it. Which I will, and we can use it as is for now.

Reply:
And as I have a "how to create a vocab PR" session with Renzo in an hour, I'd rather have this merged before that.

Reply:
Fixed